src/Router.tsx

```diff
@@ -13,23 +15,51 @@ const ImportImageWizard = lazy(
   () => import('./Components/CreateImageWizard/ImportImageWizard')
 );
 const CreateImageWizard = lazy(() => import('./Components/CreateImageWizard'));
+const isOnPremise = process.env.APP_TYPE_ENV;
+const useFlagWithAppTypeCheck = (
```

Comment:
Maybe a shorter name, like `getFlag` or `getFeatureFlag`? This name is kinda unwieldy.